en/getting-started/first-game/index.md

```diff
@@ -341,6 +341,10 @@ export class GameManager extends Component {
 }
 ```
 
+Assign the Cube prefab that made previously to the 'CubePrfb' property in GameManager Inspector.
```

Comment:
Assign the Cube prefab that made previously to the 'CubePrfb' property in GameManager Inspector.
-->
Assign the Cube prefab that made previously to the `CubePrfb` property in GameManager Inspector.